use_cases/README.md

```diff
@@ -0,0 +1,3 @@
+This directory provides examples for specific use cases. Please [open an issue](https://github.com/sendgrid/ruby-http-client/issues) or make a pull request for any use cases you would like us to document here. Thank you!
```

Comment:
@ProZsolt would this be better addressed by adding a `.github` directory containing an Issue/PR_Template? https://blog.github.com/2016-02-17-issue-and-pull-request-templates/